README.md

```diff
@@ -79,6 +83,19 @@ Additional examples can be found [here](./index.html).
 
 <img width="607" alt="Paywall" src="https://user-images.githubusercontent.com/9007851/95088220-58d0d000-072b-11eb-8cd6-57052d40765c.png">
 
+## React.js component
+
+### Usage with ES6
+
+Props the same with [plain js vesrion](#Button-(`superheroUtils.createButton`)).
+
+#### Example: 
+```js
+import TipButton from '@aeternity/superhero-utils/button-react';
+
+<TipButton size="icon" account="AK_addressThatYouWonder">
```

Comment:
```suggestion
<TipButton size="icon" account="example.chain">
```
Proposing to use the consistent data between examples, before was used `example.chain`

Reply:
Also, let's name it just Button as it named above